components/style/properties/longhand/ui.mako.rs

```diff
@@ -29,3 +29,10 @@ ${helpers.single_keyword("-moz-window-dragging", "default drag no-drag", product
                          gecko_enum_prefix="StyleWindowDragging",
                          animatable=False,
                          spec="None (Nonstandard Firefox-only property)")}
+
+${helpers.predefined_type("caret-color", 
+                          "ColorOrAuto", 
+                          "Either::Second(Auto)",
+                          spec="https://drafts.csswg.org/css-ui/#caret-color",
+                          animatable="False",
```

Comment:
For now, you can have a `products='none'` here and proceed with interpolation. Once that's done, let's change this to `products='gecko'` and we'll fix the build errors :)